include/emp/prefab/ConfigPanel.hpp

```diff
@@ -395,8 +399,30 @@ namespace prefab {
         // Otherwise val is 0 and we have nothing to go on
       }
 
-      void ExcludeConfig(const std::string setting) {
-        exclude.insert(setting);
+
+      void ExcludeConfig(const std::string & setting) {
```

Comment:
Codecov doesn’t think this function is being covered?

Reply:
I'd really like to remove this function since the naming is unclear as to what it does (sounds like you're removing the config object?) and it's undocumented, but I believe some folks are using it. I'll add documentation that recommends using `ExcludeSetting` or `ExcludeGroup` and make it clear that this does both and I'll make a test.